plugins/olm/skills/push-to-operatorhub/SKILL.md

```
@@ -0,0 +1,344 @@
+1. Clone the hive repo and both OperatorHub repos to temp directories
+2. Calculate a version name based on the branch and commit (format: `1.2.<commitcount>-<shaprefix7>`)
+3. Build and push the operator image to [quay.io/openshift-hive/hive](https://quay.io/repository/openshift-hive/hive?tab=tags)
+4. Generate OLM bundle manifests
+5. Create commits with the bundle in both OperatorHub clones
+6. Create PRs for both commits
+
+#### Step 3.2 — Capture PR URLs
+
+Extract the two PR URLs from the script output. Present them to the user:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)